Core/include/Acts/EventData/SpacePointContainer.ipp

```diff
@@ -42,8 +42,7 @@ void SpacePointContainer<container_t, holder_t>::initialize() {
     m_data.setX(i, external_container.x_impl(i) - m_options.beamPos[0]);
     m_data.setY(i, external_container.y_impl(i) - m_options.beamPos[1]);
     m_data.setZ(i, external_container.z_impl(i));
-    m_data.setRadius(
-        i, std::sqrt(m_data.x(i) * m_data.x(i) + m_data.y(i) * m_data.y(i)));
+    m_data.setRadius(i, std::hypot(m_data.x(i), m_data.y(i)));
```

Comment:
`hypot` is significantly slower than `sqrt` see https://github.com/acts-project/acts/pull/3694. I also add some helpers there

Reply:
ok, good to know !